File: 프로그래머스/0/181911. 부분 문자열 이어 붙여 문자열 만들기/부분 문자열 이어 붙여 문자열 만들기.js
function solution(my_strings, parts) {
   let result = [];
    for (let i = 0; i < my_strings.length; i++){
        let [start,end] = parts[i];
        let partStrings = my_strings[i].substring(start,end + 1);
        result.push(partStrings);
    }
            return result.join('');
}

File: 프로그래머스/0/181911. 부분 문자열 이어 붙여 문자열 만들기/부분 문자열 이어 붙여 문자열 만들기.py
# my_strings = ["progressive", "hamburger", "hammer", "ahocorasick"]
# parts = [[0, 4], [1, 2], [3, 5], [7, 7]] = [s, e]

def solution(my_strings, parts):
    result = ""
    
    for i in range(len(my_strings)):
        s,e = parts[i]
        # print("s랑e",s,e)
        # print("str->",my_strings[i])
        result += my_strings[i][s:e+1]
        
    return result
        
        